python/libraries/bimvee/importNumpy.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import os
+
+def importNumpy(filePathOrName, **kwargs):
+    outDict = {
+        'info': kwargs,
+        'data': {}
+    }
+    outDict['info']['filePathOrName'] = filePathOrName
+
+    events = np.load(filePathOrName)
+
+    outDict['data']['ch'] = {}
+    outDict['data']['ch']['dvs'] = {}
+    tsOffset = events[0, 0]
+    ts_to_sync = [events[:, 0]]
+
+    gt_filename = os.path.join(os.path.dirname(filePathOrName), 'boxes.npy')
+
+    if os.path.exists(gt_filename):
+        b_boxes = np.load(gt_filename)
+        tsOffset = min(tsOffset, b_boxes[0, 0])
+        ts_to_sync.append(b_boxes[:, 0])
+        outDict['data']['ch']['dvs']['b_boxes'] = {}
```

Comment:
I think it would be cleaner and more consistent if b_boxes are a datatype on parallel with dvs, frames etc. Then, a b-boxes markup is a property of a channel and you would know to apply it to appropriate datatypes in the same channel; so for example:

data: {
    'left atis dvs labelled': {
        'dvs': {...},
        ''bounding_boxes': {...}},
    'left atis aps labelled': {
        'frame': {...},
        ''bounding_boxes': {...}},
     'right atis labelled': {
        'dvs': {...},
        'frame': {...},
        ''bounding_boxes': {...}}}
So, in the example above, in the first two channels, each of dvs and aps has its own bounding boxes, but in the third example, there is a channel with both dvs and frames, so the same bounding boxes would be applied to both of them - more relevant in davis data where dvs and aps are completely aligned. Consider, for example, how the pixelwise object masks in https://better-flow.github.io/evimo/ could be reduced to bounding boxes. You might further talk to @lsrosa , who is currently labelling data (though not with bounding boxes or python) and think ahead about how else this could be generalised.